x-pack/plugins/observability/public/pages/alerts/parse_alert.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  RULE_ID,
+  RULE_NAME,
+  ALERT_STATUS,
+  ALERT_START,
+} from '@kbn/rule-data-utils/target/technical_field_names';
+import type { TopAlert } from '.';
+import { parseTechnicalFields } from '../../../../rule_registry/common/parse_technical_fields';
+import { asDuration, asPercent } from '../../../common/utils/formatters';
+import { ObservabilityRuleTypeRegistry } from '../../rules/create_observability_rule_type_registry';
+
+export const parseAlert = (observabilityRuleTypeRegistry: ObservabilityRuleTypeRegistry) => (
+  alert: Record<string, unknown>
```

Comment:
This is the successor to `decorateResponse`, whose argument type `TopAlertResponse` was only every indirectly defined via the return type of the server-side route registration. The removal of that route meant the type went away as well. The data fetching now happens via the `timeline` search strategy, which doesn't provide much type safety around the data, which is reflected by this `Record<string, unknown>`.